tests/beaker_tests/bgpneighboraf/test_bgpneighborafl2vpn.rb

```diff
@@ -0,0 +1,380 @@
+###############################################################################
```

Comment:
I think it makes more sense to just add these test cases to test_bgpneighboraf.rb instead of creating a new file.